src/erasure-code/jerasure/jerasure_init.cc

```diff
@@ -18,13 +18,15 @@
 #include "common/debug.h"
 #include "jerasure_init.h"
 
+#ifdef __cplusplus
 extern "C" {
+#endif
+
 #include "galois.h"
-}
 
 #define dout_context g_ceph_context
 
-extern "C" int jerasure_init(int count, int *words)
+int jerasure_init(int count, int *words)
```

Comment:
i don't understand why we need this fix or what this fix addresses. please elaborate in the commit message.